src/styles/card.css

```diff
@@ -0,0 +1,108 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.card__photo {
+  margin: 0 auto 40px;
+  
+  display: block;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
```

Comment:
You will use this color a lot, so you could move it to the variables. (actually, all the colors on the page)